packages/react/src/utilities/hooks/useIsSmallScreen.ts

```diff
@@ -0,0 +1,20 @@
+import { useCallback, useEffect, useState } from "react";
+
+export const useIsSmallScreen = (breakPointToCheck: number) => {
+  const [isSmallScreen, setIsSmallScreen] = useState<boolean>(
+    window.innerWidth < breakPointToCheck,
+  );
+
+  const handler = useCallback(() => {
+    setIsSmallScreen(window.innerWidth < breakPointToCheck);
+  }, [breakPointToCheck]);
+
+  useEffect(() => {
+    window.addEventListener("resize", handler);
+    return () => {
+      window.removeEventListener("resize", handler);
+    };
+  }, [breakPointToCheck, handler]);
```

Comment:
ici le breakPointToCheck ne fait pas parti du useEffect donc ne doit pas faire partie des dépendence.
```suggestion
  }, [handler]);
```